src/scripts/header/scroll-handler.ts

```
@@ -0,0 +1,75 @@
+
+import { getElementById, addClass, removeClass } from "../utils/dom.js";
+
+/**
+ * Gestionnaire de défilement pour le header
+ * Gère l'opacité du header en fonction de la position de défilement
+ * @class ScrollHandler
+ */
+export class ScrollHandler {
+  private desktopHeader: HTMLElement | null;
+  private mobileHeader: HTMLElement | null;
+  private lastScrollY: number = 0;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)